pvlib/pvsystem.py

```diff
@@ -2872,45 +2872,47 @@ def dc_ohms_from_percent(vmp_ref, imp_ref, dc_ohmic_percent,
                          modules_per_string=1,
                          strings=1):
     """
```

Comment:
`\times` is not rendering correctly in the docs. Using a raw string for the docstring should fix that. Same goes for `dc_ohmic_losses` but I can't edit that line
```suggestion
    r"""
```